charts/ziti-router/values.yaml

```diff
@@ -3,12 +3,18 @@ ctrl:
   # -- required control plane endpoint
   endpoint: # ctrl.example.com:6262
 
+# Explicit proxy setting in the router configuration. Router can be deployed in a site where all egress traffic is forwarded through an explicit proxy.
```

Comment:
Use the special `# --` prefix before a comment that documents an input value and the `helm-docs` job will add those explanations to the README automatically.